src/main/java/goodgenerator/blocks/tileEntity/EssentiaHatch.java

```diff
@@ -87,7 +87,7 @@ public void updateEntity() {
     }
 
     public void fillfrompipe() {
-        if (getEssentiaAmount(null) >= 1000) return;
+        if (getEssentiaAmount(null) >= 50) return;
```

Comment:
Not a huge fan of dropping from 1k internal storage to just 50, if you're going to drop to a low number, at least do a multiple of 2 like 64 or 128 to stay consistent with jars

Reply:
Maybe 32? This just limits Golem

Reply:
64 is the lowest I can do, it needs to at least be a jar